packages/runtime/src/runtime.ts

```diff
@@ -380,6 +406,7 @@ export class Runtime {
 
     account.deleteApp(appId);
     this.store.globalApps.delete(appId);
+    this.programMap.delete(appId);
```

Comment:
hmmm, i see that we're storing application data in multiple maps. Can we use a single map? Maybe rename `globalApps` to `appData`/ `appConfig` and store:
```
  appData: Map[ appId => {
    address: addr // StoreAccount address
    approval: approvalProgram // application's approval program
    clear: clearProgram // application's clear program
    ... // more data
  }]
  ```

Reply:
thanks for suggestion